### PR TITLE
Change param `using` of join() like pandas

### PR DESCRIPTION
**Pull request by xuebinsu**
Change param for the `USING` clause to `on` so that it looks
more like pandas.

### CODE REVIEW

File: greenplumpython/dataframe.py [hunk 1]
```diff
@@ -170,66 +170,83 @@ def __getitem__(self, _):
...
         ret = list(self)
+        if len(ret) == 0:
+            return "<table><tr><th></th></tr></table>"
         if len(ret) != 0:
             repr_html_str = "<table>\n"
             repr_html_str += "\t<tr>\n"
-            repr_html_str += ("\t\t<th>{:}</th>\n" * len(ret[0])).format(
-                *((ret[0]))  # type: ignore
-            )
+            repr_html_str += ("\t\t<th>{:}</th>\n" * len(ret[0])).format(*((ret[0])))
```
Comment (yihong0618) [inline greenplumpython/dataframe.py:244]: else here maybe better?
if not ret:
    return ///
else:
   do/ // 
   return

Comment (yihong0618) [inline greenplumpython/dataframe.py:244]: maybe better:

```python
if not ret:
   return "<table><tr><th></th></tr></table>"
else:
        repr_html_str = "<table>\n"
        repr_html_str += "\t<tr>\n"	            repr_html_str += "\t<tr>\n"
        repr_html_str += ("\t\t<th>{:}</th>\n" * len(ret[0])).format(	            repr_html_str += ("\t\t<th>{:}</th>\n" * len(ret[0])).format(*((ret[0])))

```

**Review by yihong0618 — APPROVED** [review]

**Comment by yihong0618** [comment]
> @beeender @yihong0618 Conflicts resolved.

dill problem, test failed

**Review by yihong0618 — APPROVED** [review]
LGTm

**Review by beeender — APPROVED** [review]

**Pull request merged**